src/BankID.php

```diff
@@ -45,6 +47,14 @@ public function __construct($environment = self::ENVIRONMENT_TEST, $certificate
             ],
         ];
 
+        if (!is_null($key)) {
+            $httpOptions['ssl_key'] = $key;
+        }
+
+        if (!is_null($passphrase)) {
```

Comment:
Some  small nitpicks: I'd suggest moving this block of code further up, and assigning it to `$certificate`:

```php
if (! $certificate) {
    $certificate = __DIR__.'/../certs/test.pem';
}

if (! is_null($passphrase)) {
    $certificate = [$certificate, $passphrase];
}
```
   
Another nit-pick: this repo uses spaces after `!`

And lastly: I noticed that Guzzle and Curl seems to use password instead of passphrase thoughout their documentation, so maybe change to `$password`?

Reply:
I changed my mind regarding passphrase/password when seeing passphrase in the BankID documentation, so let's stick with passphrase.